wiki/Legal/Copyright/Music_Licensing/en.md

```diff
@@ -0,0 +1,19 @@
+# Music Licensing
+
+The osu! team regularly reaches out to artists that the community loves and new artists whose music is appropriate for use within the community and game. Once an agreement is established, artists are listed under the [Featured Artists](/wiki/Featured_Artists) program.
```

Comment:
> Once an agreement is established

an agreement to what? the first sentence only mentions who you agree with, not what you agree to (might be obvious to most readers, but language is important on these kinds of pages)

the "what" (licensing music) should be your first sentence